cli_tools/common/gcsfuse/client.go

```diff
@@ -43,7 +43,7 @@ func (client defaultClient) MountToTemp(ctx context.Context, bucket string) (str
 	if err != nil {
 		return "", fmt.Errorf("failed to create a destination directory: %w", err)
 	}
-	cmd := exec.CommandContext(ctx, "gcsfuse", bucket, dir)
+	cmd := exec.CommandContext(ctx, "gcsfuse", "--implicit-dirs", bucket, dir)
```

Comment:
Can you add some unit tests that confirm the bug, and then show that your fix address the isssue?

Reply:
Talked offline, and Tony pointed out that gcsfuse would be difficult for unit tests. I suggested creating a module test similar to [OS inspect](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/5bd10d7493e35f6fb3ab3ed440f56ece695568b4/cli_tools_tests/module/diskinspect/disk_inspect_test.go#L341-L340)

In the test, call `NewGCSInspector`, then run `Inspect` with a gcs path, and assert the results are what you'd expect.

Reply:
added a new file for that